### PR TITLE
Branch 22.06 nvcomp notice binary [skip ci]

### PR DESCRIPTION
**Pull request by sameerz**
In the 22.06 release, we will bundle the nvcomp2.3 binary.  We need to include the associated license information in our NOTICE-binary file.  Adding that text in this PR.

### CODE REVIEW

**Review by jlowe — COMMENTED** [review]
Looks OK but need to resolve merge conflict.

File: NOTICE-binary [hunk 1]
```diff
@@ -52,3 +52,363 @@ PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
...
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+---------------------------------------------------------------------
+This software includes the nvCOMP SDK with the following licenses:
+
+LICENSE AGREEMENT FOR NVIDIA SOFTWARE DEVELOPMENT KITS
+
+This license agreement("Agreement”) is a legal agreement between you and NVIDIA Corporation
+("NVIDIA") and governs your use of the NVIDIA nvCOMP software development kit as available at
+NVIDIA’s discretion (each, a “SDK”).
```
Comment (jbrennan333) [inline NOTICE-binary:63]: (each, a "SDK") looks like a typo?

**Review by jlowe — APPROVED** [review]

File: NOTICE-binary (hunk 1 above)
Comment (sameerz) [reply NOTICE-binary:63]: I've contacted the license author and am waiting for an update.  In the meantime I'd like to merge in what we have so far.

Comment (jbrennan333) [reply NOTICE-binary:63]: That's OK with me.

**Review by jbrennan333 — APPROVED** [review]

**Pull request merged**